ostd/src/arch/x86/iommu/registers/mod.rs

```diff
@@ -165,15 +166,16 @@ impl IommuRegisters {
             "IOMMU extend capability:{:#x?}",
             iommu_regs.extended_capability()
         );
+        builder.remove(base_address as usize..(base_address as usize + PAGE_SIZE));
```

Comment:
This does not seem to be related to the three `debug!` lines, perhaps it is better to move it up (e.g. just after computing `base_address`).